scripts/run.zsh

```diff
@@ -97,6 +98,20 @@ for arg in "${command[@]}"; do
       latest_block=$(curl -s --data-raw '{"jsonrpc":"2.0","method":"eth_getBlockByNumber", "params":["latest"], "id":0}' $rpc_endpoint | jq .)
       echo "Latest Block: $latest_block"
       ;;
+    "get_slot")
+      if [[ -z "${command[2]}" ]]; then
+        echo "Please provide a slot number as the second argument, or get the latest slot"
+        echo "  Example: ${0} get_slot 100"
+        ${0} latest_slot
+        exit;
+      else
+        slot=${command[2]}
+        # Get the latest slot of the network
```

Comment:
this comment is not right. It will get a specific slot that you use as a variable.

Reply:
ehh isn't the correct? i want to get the slot i specify and default is latest :D
updated comment